style.css

```diff
@@ -0,0 +1 @@
+h1 { color: blue; }
```

Comment:
- [ ] You are using an element selector to style the heading, Please use a class selector instead to follow the project requirements. You can add to class to the heading and use that class here to style the heading.